torchao/utils.py

```diff
@@ -1,46 +1,46 @@
-import torch
+import torch
 import torch.utils.benchmark as benchmark
-
-
-def benchmark_model(model, num_runs, input_tensor):
-    torch.cuda.synchronize()
-    start_event = torch.cuda.Event(enable_timing=True)
-    end_event = torch.cuda.Event(enable_timing=True)
-    start_event.record()
-
-    # benchmark
-    for _ in range(num_runs):
-        with torch.autograd.profiler.record_function("timed region"):
-            model(input_tensor)
-
-    end_event.record()
-    torch.cuda.synchronize()
-    return start_event.elapsed_time(end_event) / num_runs
-
-def profiler_runner(path, fn, *args, **kwargs):
-    with torch.profiler.profile(
-            activities=[torch.profiler.ProfilerActivity.CPU,
-                        torch.profiler.ProfilerActivity.CUDA],
-            record_shapes=True) as prof:
-        result = fn(*args, **kwargs)
-    prof.export_chrome_trace(path)
-    return result
-
-def get_compute_capability():
-    if torch.cuda.is_available():
-        capability = torch.cuda.get_device_capability()
-        return float(f"{capability[0]}.{capability[1]}")
-    return 0.0
-
-def skip_if_compute_capability_less_than(min_capability):
-    import unittest
-    def decorator(test_func):
-        def wrapper(*args, **kwargs):
-            if get_compute_capability() < min_capability:
-                raise unittest.SkipTest(f"Compute capability is less than {min_capability}")
-            return test_func(*args, **kwargs)
-        return wrapper
-    return decorator
+
+
+def benchmark_model(model, num_runs, input_tensor):
```

Comment:
these look like linting changes? can't quite see the difference

Reply:
Maybe change in end of line symbols? iirc, Windows use different symbols for end of line.